src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -428,7 +428,7 @@ def test_iot_hub(self, resource_group, resource_group_location, storage_account)
                          self.check('properties.enableDataResidency', True)])
         self.cmd('az iot hub delete -n {}'.format(dr_hub_name))
 
-    @unittest.skip('Will be recorded in https://github.com/Azure/azure-cli/pull/22262')
+    # @unittest.skip('Will be recorded in https://github.com/Azure/azure-cli/pull/22262')
```

Comment:
A quick question, can this test be live-run by anyone on any subscription? If not, we should mark it as live-only.

Reply:
This should be runnable by anyone (as long as they have permissions to assign identities to the hub)